instrumentation/trilogy/opentelemetry-instrumentation-trilogy.gemspec

```diff
@@ -26,7 +26,16 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 3.0'
 
   spec.add_dependency 'opentelemetry-api', '~> 1.0'
-  spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.22.1'
+  # TODO: Add base version constraint back to gemspec
+  # Commented out because the following error is thrown while bundling:
+  # Because opentelemetry-instrumentation-base >= 0.22.1, < 0.23.A could not be found in source at `../base`
+  #   and every version of opentelemetry-instrumentation-trilogy depends on opentelemetry-instrumentation-base ~> 0.22.1,
+  #   opentelemetry-instrumentation-trilogy cannot be used.
+  # So, because Gemfile depends on opentelemetry-instrumentation-trilogy >= 0,
+  #   version solving has failed.
+  spec.add_dependency 'opentelemetry-instrumentation-base' # , '~> 0.22.1'
+  spec.add_dependency 'opentelemetry-instrumentation-helpers-mysql'
+  spec.add_dependency 'opentelemetry-instrumentation-helpers-sql-obfuscation'
   spec.add_dependency 'opentelemetry-semantic_conventions', '>= 1.8.0'
```

Comment:
@ericmustin  - Do you know why Trilogy requires semantic_conventions, but MySql2 doesn't? As far as I can tell, the `semantic_conventions` gem isn't being used in Trilogy. However, I'm still pretty new to the way OTel organizes things and could be missing something.

Reply:
offhand i have no idea...checking